src/controller.rs

```diff
@@ -124,12 +123,18 @@ impl MetricsController {
         }
     }
 
+    pub fn start(&self) {
+        self.logger.init();
+    }
+
```

Comment:
So, if the app calls this more than once, they all get the same instance since it's a static, right?